code/src/components/Album.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Icons } from "components/Icons"
+
+export const Album = (props) => {
+    return (
+        <div className='artistBox'>
+            <div className='album-cover'>
+                <img className='album-img' src={props.image} alt='Album' />
+                <Icons />
+            </div>
+            <div className='album-info'>
+                <div className='album-title'>
+                    <a href={props.albumUrl}>{props.albumTitle}</a>
+                    <div className='artist-name'>
+                        <a href={props.urlArtistName}>{props.artists}</a>
+                    </div>
```

Comment:
Here, to be able to show multiple artists (if there are multiple artists on the album) do a similar solution as you did in App.js to get all the albums. 
```
{props.artists.map(artist => (
    <a className="artist" href={artist.external_urls.spotify}>{artist.name}</a>
))}